test/model/recent_dm_conversations_test.dart

```diff
@@ -35,7 +36,8 @@ void main() {
             key([]):     200,
             key([1]):    100,
           })
-          ..sorted.deepEquals([key([1, 2]), key([]), key([1])]);
+          ..sorted.deepEquals([key([1, 2]), key([]), key([1])])
+          ..latestMessagesByRecipientForTesting.deepEquals({1: 300, 2: 300});
```

Comment:
It's good to update the existing test cases for this code to check the new data structure.

But are there any new test cases that become useful based on thinking through edge cases of the new data structure? I'd guess there would be. It'd then be good to add those too.

Reply:
I thought a lot about it, but frankly I couldn't come up with other edge cases!🙂 I would appreciate any clue from your side!